code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -217,6 +217,9 @@ meteor_act
 	if(W.damtype != BRUTE)
 		return
 
+	if(!should_have_organ(BP_HEART))
+		return
```

Comment:
Is this a good proxy for bloodiness across commonly used species? I seem to recall mechanical species using blood overlays to hack in oil spills and stuff.

Reply:
It's a standard check for other species code involving blood, dating back to one of the foundational organ PRs. Should be changed to its own proc most likely but that would be a broader refactor.

Reply:
fair enough.

Reply:
Yeah, I used it specifically because other similar code did.